vnext/Microsoft.ReactNative/Views/SwitchViewManager.cpp

```diff
@@ -102,6 +103,20 @@ void SwitchShadowNode::updateProperties(const folly::dynamic &&props) {
   m_updating = false;
 }
 
+void SwitchShadowNode::dispatchCommand(const std::string &commandId, const folly::dynamic &commandArgs) {
+  if (commandId == "setValue") {
+    auto value = commandArgs[0].asBool();
+    auto toggleSwitch = GetView().as<winrt::ToggleSwitch>();
+    if (toggleSwitch == nullptr)
```

Comment:
nit: `com_ptr::as` will fail-fast (terminate) if it fails to QI, so there's no need for a null-check here. The alternative would be to change `as` to `try_as`, but I think fail-fast makes sense here.

Reply:
Does that mean other code in the same file has the same nit (in `UpdateThumbColor` and `UpdateTrackColor`)?

Reply:
Yeah, it looks like they do the same thing as well 😕. The check is relatively harmless, but not necessary. Feel free to keep or remove.